src/reps/grip-map.js

```diff
@@ -157,35 +148,42 @@ const GripMap = React.createClass({
       }, []);
   },
 
+  safeObjectLink: function (config, ...children) {
+    if (this.props.objectLink) {
+      return this.props.objectLink(Object.assign({
+        object: this.props.object
+      }, config));
+    }
+
+    if (Object.keys(config).length === 0 && children.length === 1) {
+      return children[0];
+    }
+
+    return span(config, ...children);
+  },
+
   render: wrapRender(function () {
     let object = this.props.object;
     let props = this.safeEntriesIterator(object,
       (this.props.mode === MODE.LONG) ? 10 : 3);
 
-    let objectLink = this.props.objectLink || span;
     if (this.props.mode === MODE.TINY) {
       return (
         span({className: "objectBox objectBox-object"},
-          this.getTitle(object),
-          objectLink({
-            className: "objectLeftBrace",
-            object: object
-          }, "")
+          this.getTitle(object)
```

Comment:
Ah right! Somehow I thought that objectLeftBrace would add a pseudo el with the left brace as content, but it's not the case at all :)